src/standalone/Vegetation/photosynthesis_farquhar.jl

```diff
@@ -462,45 +462,25 @@ end
 ## For interfacing with ClimaParams
 
 """
-    function FarquharParameters(
-        ::Type{FT},
-        is_c3::Union{FT, ClimaCore.Fields.Field};
-        Vcmax25 = FT(5e-5),
-        kwargs...  # For individual parameter overrides
-    )
-
     function FarquharParameters(
         toml_dict::CP.ParamDict,
-        is_c3::Union{AbstractFloat, ClimaCore.Fields.Field};
-        Vcmax25 = FT(5e-5),
-        kwargs...  # For individual parameter overrides
+        is_c3::Union{AbstractFloat, ClimaCore.Fields.Field} = toml_dict["is_c3"];
+        Vcmax25 = toml_dict["Vcmax25"],
     )
 
-Constructors for the FarquharParameters struct. Two variants:
-1. Pass in the float-type and retrieve parameter values from the default TOML dict.
-2. Pass in a TOML dictionary to retrieve parameter values.Possible calls:
+Constructor for the `FarquharParameters` struct.
 ```julia
-ClimaLand.Canopy.FarquharParameters(Float64, 1.0)
-# Kwarg overrides
-ClimaLand.Canopy.FarquharParameters(Float64, 1.0; Vcmax25 = 99999999, pc = 444444444)
-# TOML Dictionary:
 import ClimaParams as CP
 toml_dict = CP.create_toml_dict(Float32);
-ClimaLand.Canopy.FarquharParameters(toml_dict, 1.0f0; Vcmax25 = 99999999, pc = 444444444)
+ClimaLand.Canopy.FarquharParameters(toml_dict, 1.0f0; Vcmax25 = 99999999)
 ```
 """
-FarquharParameters(
-    ::Type{FT},
-    is_c3::Union{FT, ClimaCore.Fields.Field};
-    kwargs...,
-) where {FT <: AbstractFloat} =
-    FarquharParameters(CP.create_toml_dict(FT), is_c3; kwargs...)
-
 function FarquharParameters(
-    toml_dict::CP.ParamDict,
-    is_c3::Union{AbstractFloat, ClimaCore.Fields.Field};
-    Vcmax25 = 5e-5,
-    kwargs...,
+    toml_dict::CP.ParamDict;
+    is_c3::Union{AbstractFloat, ClimaCore.Fields.Field},
+    Vcmax25,
+    sc = toml_dict["low_water_pressure_sensitivity"],
```

Comment:
these are now in the moisture stress parameterization and should no longer be in Farquhar.

Reply:
(they were removed from the struct, I thought, in the moisture stress PR.)